lib/models/quellenreiter_app_state.dart

```diff
@@ -392,6 +393,16 @@ class QuellenreiterAppState extends ChangeNotifier {
     opponentRequests = tempRequests;
     pendingRequests = tempPending;
     playableOpponents = tempPlayableOpponents;
+    _restoreCurrentOpponent();
```

Comment:
This is not displayed well. What I did is: Extracting the method ``_restoreCurrentOpponent()`.